### PR TITLE
Improve spelling and grammar in the introduction

### PR DESCRIPTION
**Pull request by CFBaptista**
This PR updates the spelling and grammar in the Introduction section of the Wflow documentation.
This resolves one of the tasks in issue #377, namely the update of the Introduction section.\
As I joined the team this week I have been reading the documentation in sequential order from front to back and along the way I commit languages changes.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ From 2021 the [wflow code](https://github.com/Deltares/Wflow.jl) is distributed
 [MIT License](https://github.com/Deltares/Wflow.jl/blob/master/LICENSE). Wflow is also
 available as a [compiled executable](https://download.deltares.nl/en/download/wflow/) under
 the Deltares terms and conditions. The wflow computational engine is built in the
-[Julia](https://julialang.org/) language, a high-performance computing language and does not
-have a graphical user interface, being designed for maximum user flexibility. Prior to 2021,
-wflow was developed in PCRaster Python language and is [still
-available](https://github.com/openstreams/wflow) but not actively developed.
+[Julia](https://julialang.org/) language, a high-performance computing language.
+The code does not include a graphical user interface and is designed for maximum user
```
Comment (verseve) [inline docs/src/index.md:31]:
```suggestion
Wflow does not include a graphical user interface and is designed for maximum user
```

File: docs/src/intro/use_cases.md [hunk 2]
```diff
@@ -18,51 +18,50 @@ the floodplains, which will likely improve extreme streamflow predictions.
 ## [Operational flood forecasting in Australia](@id case_flifs)
 
 In Australia, there was a need for high-resolution, fast and accurate rainfall-runoff models to
-provide boundary conditions for a fast and detailled flood inundation model (SFINCS). The
-domain of the flood model covers there complete North and East coast of Australia. Although
-many observations stations are available to provide real-time information, many rivers are not
+provide boundary conditions for a fast and detailed flood inundation model (SFINCS). The
+domain of the flood model covers the complete North and East Coast of Australia. Although
+many observation stations are available to provide real-time information, many rivers are not
```
Comment (verseve) [inline docs/src/intro/use_cases.md:23]:
```suggestion
many gauging stations are available to provide real-time information, many rivers are not
```

File: docs/src/intro/use_cases.md [hunk 3]
```diff
@@ -18,51 +18,50 @@ the floodplains, which will likely improve extreme streamflow predictions.
...
-including several small and big reservoirs. We could also demonstrate the potential for further
-calibration by changing the KsatHorFrac parameter.
+run the models operationally under the Delft-FEWS platform. Model validation was carried
+out for two basins by comparing the results of wflow\_sbm against the observations and the
+results of calibrated URBS models. This validation showed that the uncalibrated wflow\_sbm
+model results were already quite satisfying, especially given the complex nature of these
+basins, including several small and big reservoirs. We could also demonstrate the potential
+for further calibration by changing the KsatHorFrac parameter.
 
 Reference: De Kleermaeker, S., Maguire, S., Druery, C., Morales, Y., Lijnse, T., Hegnauer, M.,
```
Comment (verseve) [inline docs/src/intro/use_cases.md:43]:
```suggestion
Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.,
```

File: docs/src/intro/use_cases.md [hunk 4]
```diff
@@ -18,51 +18,50 @@ the floodplains, which will likely improve extreme streamflow predictions.
...
-2022 (in preperation). Developing a real-time data and modelling framework for operational
+2022 (in preperation). Developing a real-time data and modeling framework for operational
 flood inundation forecasting in Australia. Hydrology Water and Resources Symposium.
```
Comment (verseve) [inline docs/src/intro/use_cases.md:45]: This reference is now available (not in prep anymore), I would also stick to the original spelling of a reference (modelling).

```suggestion
2022. Developing a real-time data and modelling framework for operational flood inundation forecasting in Australia. In Hydrology & Water Resources Symposium 2022 (HWRS 2022): The Past, the Present, the Future. Engineers Australia. https://search.informit.org/doi/10.3316/informit.916755150845355
```

**Review by verseve — CHANGES_REQUESTED** [review]
Thanks for improving the spelling and grammar of the introduction!
I did add some small suggestions.

File: docs/src/index.md (hunk 1 above)
Comment (CFBaptista) [reply docs/src/index.md:31]: Done

File: docs/src/intro/use_cases.md (hunk 2 above)
Comment (CFBaptista) [reply docs/src/intro/use_cases.md:23]: Done

File: docs/src/intro/use_cases.md (hunk 4 above)
Comment (CFBaptista) [reply docs/src/intro/use_cases.md:45]: Done

File: docs/src/intro/use_cases.md [hunk 5]
```diff
@@ -40,9 +40,8 @@ model results were already quite satisfying, especially given the complex nature
 basins, including several small and big reservoirs. We could also demonstrate the potential
 for further calibration by changing the KsatHorFrac parameter.
 
-Reference: De Kleermaeker, S., Maguire, S., Druery, C., Morales, Y., Lijnse, T., Hegnauer, M.,
-2022 (in preperation). Developing a real-time data and modeling framework for operational
-flood inundation forecasting in Australia. Hydrology Water and Resources Symposium.
+Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.,
+2022. Developing a real-time data and modelling framework for operational flood inundation forecasting in Australia. In Hydrology & Water Resources Symposium 2022 (HWRS 2022): The Past, the Present, the Future. Engineers Australia. https://search.informit.org/doi/10.3316/informit.916755150845355
```
Comment (verseve) [inline docs/src/intro/use_cases.md:44]: Would be good to wrap these lines (e.g. with Rewrap (`Wrapping Column` of 92)).

**Review by verseve — APPROVED** [review]
LGTM! Only one small comment about wrapping text.

File: docs/src/intro/use_cases.md (hunk 5 above)
Comment (CFBaptista) [reply docs/src/intro/use_cases.md]: Done

**Pull request merged**